modules/config_games/server_configs/hurtworld_linux32.xml

```diff
@@ -25,4 +25,20 @@
       <installer_login>anonymous</installer_login>
     </mod>
   </mods>
+  <post_install>

```

Comment:
Just my two cents, but what if this file is updated? Wouldn't it better to copy it everytime server is updated, just in case?